packages/atomic/stencil.config.ts

```diff
@@ -180,63 +254,38 @@ export const config: Config = {
       ],
     }),
     replace(),
+    isCDN &&
+      replaceWithASTPlugin({
+        replacements: generateReplaceValues(),
+      }),
   ],
   rollupPlugins: {
     before: [
       isDevWatch &&
         alias({
-          entries: [
-            {
-              find: '@coveo/headless/case-assist',
-              replacement: path.resolve(
-                __dirname,
-                './src/external-builds/case-assist/headless.esm.js'
-              ),
-            },
-            {
-              find: '@coveo/headless/commerce',
-              replacement: path.resolve(
-                __dirname,
-                './src/external-builds/commerce/headless.esm.js'
-              ),
-            },
-            {
-              find: '@coveo/headless/recommendation',
-              replacement: path.resolve(
-                __dirname,
-                './src/external-builds/recommendation/headless.esm.js'
-              ),
-            },
-            {
-              find: '@coveo/headless/commerce',
-              replacement: path.resolve(
-                __dirname,
-                './src/external-builds/commerce/headless.esm.js'
-              ),
-            },
-            {
-              find: '@coveo/headless/insight',
-              replacement: path.resolve(
-                __dirname,
-                './src/external-builds/insight/headless.esm.js'
-              ),
-            },
-            {
-              find: '@coveo/headless',
-              replacement: path.resolve(
-                __dirname,
-                './src/external-builds/headless.esm.js'
-              ),
-            },
-          ],
+          entries: generateAliasEntries(),
         }),
       html({
         include: 'src/templates/**/*.html',
       }),
       isDevWatch && replaceHeadlessMap(),
+      externalizeDependenciesPlugin(),
```

Comment:
This is the piece of code that externalizes headless and bueno for  all builds except cdn ?

Reply:
Yes. Technically it does it for the CDN build too but since we have already replaced with the URLs it has no effect.